### PR TITLE
Isolate css framework

### PR DESCRIPTION
**Pull request by kristofferlind**
Usage of semantic-ui already scoped inside class 'qv-object-qsstatistic', which seems to be unique among the extensions.

### CODE REVIEW

File: semantic/src/semantic.less [hunk 1]
```diff
@@ -63,4 +62,4 @@
 & { @import "definitions/modules/sidebar"; }
 & { @import "definitions/modules/sticky"; }
 & { @import "definitions/modules/tab"; }
-& { @import "definitions/modules/transition"; }
+& { @import "definitions/modules/transition"; }
```
Comment (marols) [inline semantic/src/semantic.less]: Do we need that newline GitHub is complaining about?

File: semantic/src/themes/default/globals/site.overrides [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
         Global Overrides
 *******************************/
+@mini            : unit( round(@miniSize * @emSize) / @emSize, em);
+@tiny            : unit( round(@tinySize * @emSize) / @emSize, em);
+@small           : unit( round(@smallSize * @emSize) / @emSize, em);
+@medium          : unit( round(@mediumSize * @emSize) / @emSize, em);
+@large           : unit( round(@largeSize * @emSize) / @emSize, em);
+@big             : unit( round(@bigSize * @emSize) / @emSize, em);
+@huge            : unit( round(@hugeSize * @emSize) / @emSize, em);
+@massive         : unit( round(@massiveSize * @emSize) / @emSize, em);
```
Comment (marols) [inline semantic/src/themes/default/globals/site.overrides]: Are these font-sizes? Did we get them from Qlik?

Also, newline complaint.

File: semantic/src/themes/default/globals/site.variables [hunk 3]
```diff
@@ -343,14 +343,14 @@
...
-@huge            : unit( round(@hugeSize * @emSize) / @emSize, rem);
-@massive         : unit( round(@massiveSize * @emSize) / @emSize, rem);
+@mini            : unit( round(@miniSize * @emSize) / @emSize, em);
+@tiny            : unit( round(@tinySize * @emSize) / @emSize, em);
+@small           : unit( round(@smallSize * @emSize) / @emSize, em);
+@medium          : unit( round(@mediumSize * @emSize) / @emSize, em);
+@large           : unit( round(@largeSize * @emSize) / @emSize, em);
+@big             : unit( round(@bigSize * @emSize) / @emSize, em);
+@huge            : unit( round(@hugeSize * @emSize) / @emSize, em);
+@massive         : unit( round(@massiveSize * @emSize) / @emSize, em);
```
Comment (marols) [inline semantic/src/themes/default/globals/site.variables]: Looks like the same definitions as in site.overrides. Do we need both definitions?

File: README.md [hunk 4]
```diff
@@ -135,6 +135,10 @@ fa fa-calendar
 ```
 ![Font awesome](https://raw.githubusercontent.com/alner/qsStatisticBlock/screenshots/screenshots/fontawesome.png)
 
+## Data limits
+Measurements: 1-256
```
Comment (marols) [inline README.md]: Nice that you found the limitations in `definition.js` and documented them!

File: semantic/src/semantic.less (hunk 1 above)
Comment (kristofferlind) [reply semantic/src/semantic.less]: Having a newline at the end helps with diffs when content is appended, but in this case all the changes in the semantic directory is just reverting back to what was there initially as the changes resulted in some visual bugs.

File: README.md (hunk 4 above)
Comment (kristofferlind) [reply README.md]: thanks :)

**Pull request merged**